pages/products.js

```diff
@@ -0,0 +1,17 @@
+import Products from '../components/organism/Products/Products'
+
+function ProductsIndex({pageProps}) {
+  return (
+    <Products {...pageProps}/>
+  )
+}
+
+ProductsIndex.getInitialProps = async function () {
```

Comment:
IF we are required to have getInitialProps on all the pages then please move this to a HOC and wrap the pages component with HOC.